### ISSUE TITLE
bbcenter vector sign

### ISSUE THREAD

**Issue by msalmirim** [esmini/esmini#353]
Hi,

I was a bit confused when the generated bbcenter vector of a moving vehicle (car) was positive. Isn't supposed to be negative since it points from the reference point to the origin of the vehicle coordinate system (center of the middle axle projected to the ground) ? 

Thanks for your help.
Rim

**Comment by eknabevcc** [comment]
I think you refer to the center of bounding box, which should be relative the reference point of the objects - which is at center of rear axle. So I think it's correct to be positive along x-axis.

**Comment by eknabevcc** [comment]
I think you're absolutely right. Now I understand. I thought we talked about the OpenSCENARIO [bounding box center](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/Center.html) relative to vehicle reference point. But, yes, we're talking about the OSI bbcenter_to_rear attribute relative the rear axle (projected on the ground I guess). 

That should be in the opposite direction, I agree. I think also the z-value should be negated as well. And possibly y as well, but that is typically ~0 anyway.

What do you think @mguldogu and @mander76, you agree as well? In that case we should just insert a '-' to lines [739-741](https://github.com/esmini/esmini/blob/99b7f2aef94bee3533a57c103cc91ee40a268ecb/EnvironmentSimulator/Modules/ScenarioEngine/SourceFiles/OSIReporter.cpp#L739).

**Comment by mguldogu** [comment]
@eknabevcc Based on the osi definition @msalmirim stated above, I agree that this value should be negated. 
I assume this has never been realized at least by our external simulator as this value was always used only as an absolute value to perform some offset calculations. 

Let's do it as you suggest.
**In that case we should just insert a '-' to lines [739-741]** . See @rjohan69, as this requires updates from our external simulator.

**Comment by eknabevcc** [comment]
Thanks @mguldogu for input. Let's sync tomorrow to realize side effects before changing 👍

**Comment by eknabevcc** [comment]
@msalmirim and @mguldogu : "middle of the rear axle", do you think it's the actual middle of axle? I mean also z-coordinate should be the height of the axle? And not like in OpenSCENARIO where the reference point is on ground level (at normal load).

So for a example, if rear axle is on z = 0.5 and bounding box center is on z = 0.75. Then the z-component of the bbcenter_to_rear will be -0.25. Right? (while it would be -0.75 if OpenSCENARIO ref point was used)

What do you think?

**Comment by msalmirim** [comment]
Yes I agree, it got me confused as well because it's different from the OpenSCENARIO reference point but according to the definition in OSI and the graphical representation of the vector it would be the actual middle of the rear axle and the z-coordinate would be the rear axle z position  - z position of the center of the bounding box.

**Comment by eknabevcc** [comment]
Fixed. Merged to master branch. Will be included in next release.
